otherFiles/ImmutableExample.java

```diff
@@ -0,0 +1,21 @@
+import java.util.*;
```

Comment:
wildcard imports are not allowed. 
https://mate-academy.github.io/style-guides/java/java.html#s3.3.1-wildcard-imports